docs/data/material/design-resources/material-ui-sync/material-ui-sync.md

```diff
@@ -237,16 +237,13 @@ You can also check out the Storybook preview to test the Material UI version of
 ## Using the generated theme
 
 :::warning
-Themes generated by Sync must be used with Material UI's [`CssVarsProvider`](/material-ui/experimental-api/css-theme-variables/migration/)—the default [`ThemeProvider`](/material-ui/customization/theming/#theme-provider) is not supported.
+Themes generated by Sync must be used with Material UI's [`CssVarsProvider`](/material-ui/migration/migration-css-theme-variables/)—the default [`ThemeProvider`](/material-ui/customization/theming/#theme-provider) is not supported.
```

Comment:
```suggestion
Themes generated by Sync must be used with Material UI's [`CssVarsProvider`](/material-ui/migration/migration-css-theme-variables/)—the [`ThemeProvider`](/material-ui/customization/theming/#theme-provider) is not supported.
```

I wouldn't reference the `ThemeProvider` as a default.

Reply:
Out of curiosity, why not?

Reply:
@danilo-leal in v6, it's not the default provider.